config/routes.rb

```diff
@@ -35,7 +35,11 @@
   end
 
   resources :submissions, only: [:show, :update] do
-    resources :evaluations, only: [:new]
+    resources :evaluations, only: [:new] do
+      member do
+        patch 'recuse'
+      end
```

Comment:
something weird about this route is that it requires an evaluation :id param in the URL, which is why you have to add `/new/` to the route, or use a fake ID, if the evaluation doesn't exist yet. this is because you're really operating on a different underlying EvaluatorSubmissionAssignment mode rather than the possibly non-existent evaluation itself.
if you make this a collection route instead of a member route it should work the same way with one less path segment. I like the one-liner style of route for this.
```suggestion
      # route: PATCH "/submissions/3/evaluations/recuse"
      # helper: recuse_submission_evaluations_path(submission.id)
      patch 'recuse', on: :collection
```
tested this locally and it works